samples/rag-aisearch/javascript/src/app.js

```diff
@@ -20,13 +20,14 @@ app.http('IngestFile', {
         }
 
         let uri = requestBody.Url;
-        let filename = uri.split('/').pop();
+        let url = new URL(uri);
 
-        context.extraOutputs.set(embeddingsStoreOutput, { title: filename });
+        let fileName = url.pathname.split('/').pop();
```

Comment:
Using `new URL` looks good, but this is the standard way to get the file name from pathname:
```
let fileName = path.basename(url.pathname);
```


At the top of the file you'll need to add this:
- JavaScript:

    ```
    const path = require('path');
    ```

- TypeScript: 

    ```
    import * as path from 'path';
    ```

Reply:
resolved